vassal-app/src/main/java/VASSAL/counters/ActionButton.java

```diff
@@ -161,18 +161,6 @@ public HelpFile getHelpFile() {
     return HelpFile.getReferenceManualPage("ActionButton.html"); // NON-NLS
   }
 
-  // Implement Loopable
-  @Override
-  public String getComponentName() {
-    // Use inner name to prevent recursive looping when reporting errors.
-    return piece.getName();
-  }
-
-  @Override
-  public String getComponentTypeName() {
-    return getDescription();
-  }
-
```

Comment:
Why are these being removed?

Reply:
Because those two methods are now also part of the Auditable interface and are implemented in Decorator for all traits.